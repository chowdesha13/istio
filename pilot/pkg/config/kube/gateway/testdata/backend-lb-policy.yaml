apiVersion: gateway.networking.k8s.io/v1alpha2
kind: BackendLBPolicy
metadata:
  name: lb-policy
  namespace: default
spec:
  targetRefs:
  - group: core
    kind: service
    name: echo-service
  sessionPersistence:
    sessionName: foo
    absoluteTimeout: 1h
    type: Cookie
    cookieConfig:
      lifetimeType: Permanent
---