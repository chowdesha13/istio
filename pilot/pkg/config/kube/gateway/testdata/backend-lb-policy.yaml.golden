apiVersion: networking.istio.io/v1
kind: DestinationRule
metadata:
  creationTimestamp: null
  name: lb-policy
  namespace: default
spec:
  host: echo-service.default.svc.domain.suffix
  trafficPolicy:
    loadBalancer:
      consistentHash:
        httpCookie:
          name: foo
          ttl: 3600s
---